### PR TITLE
build: update required files for release-v3.3

### PR DESCRIPTION
**Pull request by Madhu-1**
updated the required templates and upgrade document for release 3.3

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

File: docs/ceph-csi-upgrade.md [hunk 1]
```diff
@@ -92,6 +94,11 @@ to upgrade from cephcsi v3.0 to v3.1
 
 ## Upgrading from v3.1 to v3.2
 
+Refer [upgrade-from-v3.1-v3.2](https://github.com/ceph/ceph-csi/blob/v3.2.1/docs/ceph-csi-upgrade.md)
+to upgrade from cephcsi v3.2 to v3.2
```
Comment (humblec) [inline docs/ceph-csi-upgrade.md]: nit: cephcsi v3.1 to v3.2

**Comment by humblec** [comment]
@Madhu-1 below change in build.env can also be made, Isnt it ?
```
-CSI_IMAGE_VERSION=canary
+CSI_IMAGE_VERSION=v3.3.0
```

**Comment by Madhu-1** [comment]
> @Madhu-1 below change in build.env can also be made, Isnt it ?
> 
> ```
> -CSI_IMAGE_VERSION=canary
> +CSI_IMAGE_VERSION=v3.3.0
> ```

its already done, or am i missing?

**Comment by humblec** [comment]
> > @Madhu-1 below change in build.env can also be made, Isnt it ?
> > ```
> > -CSI_IMAGE_VERSION=canary
> > +CSI_IMAGE_VERSION=v3.3.0
> > ```
> 
> its already done, or am i missing?

I missed it while reviewing :)

**Review by humblec — APPROVED** [review]

**Comment by humblec** [comment]
Merging this manually

**Pull request merged**